test_scripts/API/AutoCompleteList/019_ATF_SetGlobalProperties_N_AutoCompleteList_ArrayValuesOutOfUpperBound.lua

```diff
@@ -0,0 +1,72 @@
+--------------------------------------------------------------------------------------------
```

Comment:
@Aleksandriys, it is not clear, what this script is checking:
1) array value out of upper bound (this I take from script name) -if so, in array could be only one string element but with more than 50 chars;
or
2) array out of upper bound(this is from test name) - if so, you should use some constructor function for creating array in --[[ Local variables ]] and not use big constructions in requests.